openfga/v1/openfga_service.proto

```diff
@@ -774,6 +774,18 @@ service OpenFGAService {
   }
 }
 
+// Controls the consistency preferences when calling the query APIs.
+// buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX
+// buf:lint:ignore ENUM_VALUE_PREFIX
+enum ConsistencyPreference {
+  // Default if not set. Behavior will be the same as MINIMIZE_LATENCY
+  UNSPECIFIED = 0;
+  // Minimize latency at the potential expense of higher consistency.
+  MINIMIZE_LATENCY = 1;
```

Comment:
Just to give us room for future additions, maybe we can have MINIMIZE_LATENCY=100 and HIGHER_CONSISTENCY=200?